templates/al2023/provisioners/install-nvidia-driver.sh

```diff
@@ -40,6 +40,11 @@ function install-nvidia-container-toolkit() {
   done
 }
 
+# NVIDIA Drivers are on version behind in Isolated partitions.
+if is-isolated-partition; then
+  NVIDIA_DRIVER_MAJOR_VERSION = 555
```

Comment:
let's not overwrite the version here, it can be set via the packer variables for whoever runs the build